manifests/init.pp

```diff
@@ -117,5 +70,4 @@
   class { '::ntp::config': } ~>
   class { '::ntp::service': } ->
   anchor { 'ntp::end': }
```

Comment:
Shiny, but very tight version constraint. :-/  @ahpook what's your opinion on switching the puppet version requirement from >=4.5 to >=4.8?